src/main/java/org/openrewrite/github/AddCronTrigger.java

```diff
@@ -58,4 +95,68 @@ public String getDescription() {
     public Duration getEstimatedEffortPerOccurrence() {
         return Duration.ofMinutes(5);
     }
+
+    static class RandomCronExpression {
+        private final Random random;
+
+        public RandomCronExpression(Random random) {
+            this.random = random;
+        }
+
+        public int random(int min, int max) {
+            return random.nextInt(max + 1 - min) + min;
+        }
+
+        public int minute() {
+            return random(0, 59);
+        }
+
+        public int hour() {
+            return random(0, 12);
+        }
+
+        public String dayOfWeek() {
+            final List<String> daysOfWeek = Arrays.asList("mon", "tue", "wed", "thu", "fri", "sat", "sun");
+            return daysOfWeek.get(random.nextInt(daysOfWeek.size()));
+        }
+
+        public String dailyCron() {
+            return String.format("%d %d * * *", minute(), hour());
+        }
+
+        public String weeklyCron() {
+            return String.format("%d %d * * %s", minute(), hour(), dayOfWeek());
+        }
+
+        public String weekdays() {
+            return String.format("%d %d * * 1-5", minute(), hour());
+        }
+
+        public String weekends() {
+            return String.format("%d %d * * sat,sun", minute(), hour());
+        }
+
+        public String monthlyCron() {
+
+            return String.format("%d %d %s * *", minute(), hour(), dayOfTheMonth());
+        }
+
+        public int dayOfTheMonth() {
+            return random(1, 31);
```

Comment:
Now I never liked February, but this looks like we could skip that month for unlucky jobs. Any suggestions there?

Reply:
Pushed @yeikel
[Update AddCronTrigger.java](https://github.com/openrewrite/rewrite-github-actions/pull/56/commits/d19e9c0e6e489a23181c82f1cc66748d2884c7ae) to be safe

Reply:
Thanks! That's safer indeed; not an ideal spread throughout the year, but I think we both agree this is enough. ;)